logical-comaprison-operators.js

```diff
@@ -48,32 +48,49 @@ const isHappy = false;
 // - Check if num is between 10 and 20 (inclusive) using the logical AND operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
 
+
 // - Check if num is either less than 5 or greater than 50 using the logical OR operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
+let numlees = num < 5
+console.log(numlees);
 
 // - Check if str is either "apple" or "orange" using the logical OR operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
+let stror = "apple"||"orange"
+console.log(stror);
 
 // - Check if isHappy value is true using the logical NOT operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
+let isHappynot = !isHappy
+console.log(isHappynot);
 
 // - Check if num is even and greater than 10 using the logical AND operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
+let numeven = num >= 10
+console.log(numeven);
 
 // - Check if num is divisible by both 3 and 5 using the logical OR operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
 
+let numdivisible = num/3 && num/5
```

Comment:
To check if a number is divisible by another number, you should use the mod operator `%` and check if the value is equal to 0.
```
console.log( num % 3 === 0 && num % 5 === 0 )